### PR TITLE
track pageloads with google analytics. fixes #1012

### LINKED ISSUE #1012: track pageloads with GA better
add `ga('send', 'pageview');` to the code every time we use the backbonejs history pushState function, so that google analytics can track non-page-reload page changes

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

**Comment by Connoropolous** [comment]
and @devvmh as long as we like this one, this would be a good one to hotfix in to metamaps.cc so we can have more insight in the analytics sooner rather than later

File: frontend/src/Metamaps/Router.js [hunk 1]
```diff
@@ -202,6 +202,11 @@ const _Router = Backbone.Router.extend({
 
 const Router = new _Router()
 
+Router.navigateAndTrack = (fragment, options) => {
+  Router.navigate(fragment, options)
+  ga && ga('send', 'pageview')
```
Comment (devvmh) [inline frontend/src/Metamaps/Router.js]: Can you change this to `window.ga && window.ga('send', 'pageview')`? Stuff like this always causes trouble if you're in a node env (e.g. testing)

**Comment by devvmh** [comment]
LGTM other than that one comment. I've just now hotfixed this into metamaps.cc

File: frontend/src/Metamaps/Router.js (hunk 1 above)
Comment (Connoropolous) [inline frontend/src/Metamaps/Router.js]: Yep

**Pull request merged**